FWCore/ServiceRegistry/src/GlobalContext.cc

```diff
@@ -74,9 +92,15 @@ namespace edm {
       case GlobalContext::Transition::kEndRun:
         os << "global end Run " << RunID(gc.luminosityBlockID().run());
         break;
+      case GlobalContext::Transition::kEndProcessBlock:
+        os << "end ProcessBlock";
+        break;
       case GlobalContext::Transition::kEndJob:
         os << "endJob";
         break;
+      case GlobalContext::Transition::kWriteProcessBlock:
+        os << "write ProcessBlock ";
```

Comment:
There seems to be an extra space at the end of the string.

Reply:
Done.